src/utility.mjs

```diff
@@ -231,13 +231,9 @@ export function isDecorator(ident) {
   return suffixes.some((suffix) => ident.endsWith(suffix));
 }
 
-export function isPowerOfTwo(x) {
-  return x > 0 && (x & (x - 1)) == 0;
-}
```

Comment:
This may be used by user JS libraries - is it worth removing? If so, let's at least mention in the changelog?

Reply:
No, the way these "exports" work is only to other ES6 modules that make up the compiler.

The JS libraries don't have visibility of these symbols, by design.  JS libraries can only see symbols that are explictly added using `addToCompileTimeContext`.    This function (isPowerOfTwo) is not one of those.

Reply:
Oh, nice, I wasn't aware that ES6 modules limited things what way.